Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -243,8 +244,14 @@ class Connection : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Connec
     using Handle = ConnectionHandle;
 
     struct Identifier {
+        WTF_MAKE_NONCOPYABLE(Identifier);
+
         Identifier() = default;
 #if USE(UNIX_DOMAIN_SOCKETS)
+        Identifier(Identifier&& identifier)
```

Comment:
Any reason why this isn't explicit too?

Reply:
Um, it's been a while. It might be a mistake, or maybe it was important for some reason? Before landing, I will try making it explicit and will find out.

Reply:
Doesn't matter, because Chris's suggestion to use default constructor worked.

Reply:
Er, well at least it works on Linux, but I think it will work everywhere.